thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -2,7 +2,17 @@ import Ember from 'ember';
 import moment from 'moment';
 import d3 from 'd3';
 import buildTooltip from 'thirdeye-frontend/helpers/build-tooltip';
-import { toBaselineUrn, toMetricUrn, filterPrefix, hasPrefix, stripTail, toBaselineRange, toFilters, toMetricLabel } from 'thirdeye-frontend/helpers/utils';
+import {
+  getColor,
+  toBaselineUrn,
+  toMetricUrn,
+  filterPrefix,
+  hasPrefix,
+  stripTail,
+  toBaselineRange,
+  toFilters,
+  toMetricLabel
+} from 'thirdeye-frontend/helpers/utils';
```

Comment:
i wonder if there's a better way of doing this throughout our project

Reply:
We could also import the whole utils as 'utils' and use them in the code as utils.toFilters etc...